src/utils/calculateWidth.test.js

```diff
@@ -2,18 +2,18 @@ import calculateWidth from './calculateWidth';
 
 describe('Tooltip offset calculator', () => {
   it('should calculate width for a 1 hour meeting', () => {
-    expect(calculateWidth(1)).toBe(80);
+    expect(calculateWidth(1)).toBe(79);
```

Comment:
I had to change the calculate width function. Before each meeting had a 2px right border which was making every meeting look strange on the timeline. We now subtract 3 from the calculated width instead of 2 to prevent the meeting from covering up the markers that denote the hours on the timeline.